test/Serialization/upgrades/setup_tests.jl

```diff
@@ -6,7 +6,7 @@ if !isdefined(Main, :serialization_upgrade_test_path) ||
   !isdir(Main.serialization_upgrade_test_path) ||
   !isfile(joinpath(Main.serialization_upgrade_test_path, "LICENSE.md"))
 
-  serialization_upgrade_test_path = let commit_hash = "90c84f6b03d22d60cee6db5a55879456d7def9c2"
+  serialization_upgrade_test_path = let commit_hash = "051f76410adb7cefdb30fa8238c5c28434cf7720"
```

Comment:
```suggestion
  serialization_upgrade_test_path = let commit_hash = "81db50792055d6d44b645acf69f8a9b1f045eb8a"
```
I just merged https://github.com/oscar-system/serialization-upgrade-tests/pull/11